cdc/kv/client.go

```diff
@@ -53,11 +53,12 @@ import (
 )
 
 const (
-	dialTimeout               = 10 * time.Second
-	maxRetry                  = 100
-	tikvRequestMaxBackoff     = 20000   // Maximum total sleep time(in ms)
-	grpcInitialWindowSize     = 1 << 27 // 128 MB The value for initial window size on a stream
-	grpcInitialConnWindowSize = 1 << 27 // 128 MB The value for initial window size on a connection
+	dialTimeout           = 10 * time.Second
+	maxRetry              = 100
+	tikvRequestMaxBackoff = 20000 // Maximum total sleep time(in ms)
+	// TODO find optimal values here
+	grpcInitialWindowSize     = 1 << 28 // 128 MB The value for initial window size on a stream
+	grpcInitialConnWindowSize = 1 << 29 // 128 MB The value for initial window size on a connection
```

Comment:
comment need to be update, 1 << 28 should be 256, 1 << 29 is 512

Reply:
Yes. These are just experimental values, will need to be finalized before releasing.

Reply:
ping @liuzix